source/libraries/ContractInterfaces.ts

```diff
@@ -523,6 +523,13 @@ export class Controller extends Contract {
         return <Array<string>>result;
     }
 
+    public getTimestamp_ = async( options?: { sender?: string }): Promise<BN> => {
+        options = options || {};
+        const abi: AbiFunction = {"constant":true,"inputs":[],"name":"getTimestamp","outputs":[{"name":"","type":"uint256"}],"payable":false,"stateMutability":"view","type":"function"};
+        const result = await this.localCall(abi, [], options.sender);
+        return <BN>result[0];
+    }
```

Comment:
I would have expected to see the addition of `Time` and `TimeControlled` contracts to this file, are they being ignored for some reason?

Reply:
Hm good point. They should be added. Ill re run the generator